<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777214</width>
        <height>16777215</height>
       </size>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>291</width>
         <height>549</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="checkBox1">
          <property name="text">
           <string>Show grid</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox2">
          <property name="text">
           <string>Triangle filling</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox3">
          <property name="text">
           <string>Default texture</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxNormalMap">
          <property name="text">
           <string>Default Normal Map</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxLight">
          <property name="text">
           <string>Light Animation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxTurnOffLight">
          <property name="text">
           <string>Turn Off Light</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxRefectors">
          <property name="text">
           <string>Turn On Reflectors</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="colorButton">
          <property name="text">
           <string>Select Grid Color</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="colorLightButton">
          <property name="text">
           <string>Select Light Color</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fileButton">
          <property name="text">
           <string>Load Texture</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteButton">
          <property name="text">
           <string>Delete Texture</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="normalMapLoadButton">
          <property name="text">
           <string>Load Normal Map</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteNormalMapButton">
          <property name="text">
           <string>Delete Normal Map</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label1">
            <property name="text">
             <string>Grid precision:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider1">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label2">
            <property name="text">
             <string>Alpha angle:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label3">
            <property name="text">
             <string>Betha angle:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider3">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="labelKd">
            <property name="text">
             <string>Dispersion:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sliderKd">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLabel" name="labelKs">
            <property name="text">
             <string>Refelctance:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sliderKs">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QLabel" name="labelM">
            <property name="text">
             <string>Gloss:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sliderM">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QLabel" name="labelZ">
            <property name="text">
             <string>Light coord (z):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sliderZ">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="labelML">
            <property name="text">
             <string>mL:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSliderML">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
